spring-boot-project/spring-boot-docs/src/docs/asciidoc/data/nosql.adoc

```diff
@@ -41,6 +41,41 @@ The following listing shows an example of such a bean:
 include::{docs-java}/features/nosql/redis/connecting/MyBean.java[]
 ----
 
+You can set the configprop:spring.data.redis.url[] property to change the URL and configure
+additional settings as shown in the following example:
+
+[source,yaml,indent=0,subs="verbatim",configprops,configblocks]
+----
+	spring:
+	  data:
+	    redis:
+	      url: "redis://user:secret@redis.example.com:6399
+----
+
+Alternatively, you can specify connection details using discrete properties. For example, you might
+declare the following settings in your `application.yml`:
+
+[source,yaml,indent=0,subs="verbatim",configprops,configblocks]
+----
+	spring:
+	  data:
+	    redis:
+	      host: "redis.example.com"
+	      port: 6399
+	      username: "user"
+	      password: "secret"
+----
+When using both the URL and the discrete properties, any values that are set in the URL will take precedence over the values set in the properties.
+
+The url property supports the `redis` and `rediss` schemes as well as the following additional schemes when using Lettuce:
+
+* `redis+ssl`
```

Comment:
The list of schemes we support is only limited by what the Lettuce [RedisURI](https://github.com/lettuce-io/lettuce-core/blob/b161915eefbc1ea92131c65086787230ea227253/src/main/java/io/lettuce/core/RedisURI.java#L143). I was hesitant to link to that detail here though and instead list out the schemes directly.